docker-compose.yaml

```diff
@@ -82,16 +82,29 @@ services:
       - LDAP_USER=${LDAP_USER}
       - LDAP_PASS=${LDAP_PASS}
       - LOGGING_LEVEL=${LOGGING_LEVEL:-INFO}
-      - NEXT_ADMIN_NAME=${NEXT_ADMIN_NAME}
-      - NEXT_ADMIN_USER=${NEXT_ADMIN_USER}
-      - NEXT_ADMIN_PASS=${NEXT_ADMIN_PASS}
-      - NEXT_ADMIN_EMAIL=${NEXT_ADMIN_EMAIL}
       - USER_BASE_DN=${USER_BASE_DN}
       - GROUP_BASE_DN=${GROUP_BASE_DN}
     depends_on:
       - ledger-sync
     command: ./bin/rbac-providers-ldap
 
+  rbac-bootstrap-worker:
+    build:
+      context: .
+      dockerfile: ./docker/bootstrap.Dockerfile
+    container_name: rbac-bootstrap-worker
+    image: rbac-bootstrap-worker:${ISOLATION_ID-latest}
+    volumes:
+      - ".:/project/hyperledger-rbac"
+    environment:
```

Comment:
Some logs aren't showing up for this container because we need to pass in environment variable `LOGGING_LEVEL` like in the other containers.